### PR TITLE
Added directory search to DLS

### PR DESCRIPTION
**Pull request by lalov1**
I could add more people if it looks weird with the same people.

### CODE REVIEW

**Review by Uri-E — APPROVED** [review]
Looks pretty gewd!

**Review by prd409-utsa — APPROVED** [review]

**Comment by garza-utsa** [comment]
make sure an conform to the profile-group--compact variant:

https://utsa-asc.github.io/college-dls/components/detail/profile-group--compact.html

**Comment by lalov1** [comment]
> make sure an conform to the profile-group--compact variant:
> 
> https://utsa-asc.github.io/college-dls/components/detail/profile-group--compact.html
> 
> I think this just needs the wrapping .profile-card-list-compact inside the container

Maybe I'm confused, but I thought it would bring in the header and link if I did that.

**Comment by garza-utsa** [comment]
the .profile-card-list just has override for the .row>div and .profile-card > img heights, so you should be good if you just add that class to the wrapper like the --compact.hbs does here, no need to render the entire compact element:
https://github.com/utsa-asc/college-dls/blob/main/components/03-sections/05-content-group/06-profile-group/profile-group--compact.hbs#L3

Altho, one might argue to keep it atomic, should we have a header/link less variant (of profile group) for search to use?

File: components/04-reference-pages/16-faculty-directory-search/faculty-directory-search--compact.hbs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  <main>
+ <!--Slider -->
+    <section class="slider-wrapper" id="main-slider-wrapper">
+      {{> @banner--slim}}
+      {{> @breadcrumb }}
+    </section>
+    <!-- End Slider -->
+    <!--Begin Form Element-->
+    <section class="py-5">
+      <div class="container">
```
Comment (garza-utsa) [inline components/04-reference-pages/16-faculty-directory-search/faculty-directory-search--compact.hbs:19]: you should be able to add the .profile-card-list-compact here

**Review by garza-utsa — CHANGES_REQUESTED** [review]
see class below on your container wrapper

**Pull request merged**